accessibility/house-style.md

```diff
@@ -0,0 +1,125 @@
+
+Accessibility supports social inclusion for people with disabilities as well as others, such as older people, people in rural areas, and people in developing countries.
+
+
+### Why do we care about accessibility?
+
+- *It's a legal requirement:* we're legally obliged to make our content accessible to users with diverse needs. All of our websites MUST comply with [US Section 508](https://www.section508.gov/) of the US Rehabilitation Act 1973, [EU Directive 2016/2102](http://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32016L2102) and the [UK Equality Act 2010](http://www.legislation.gov.uk/ukpga/2010/15/contents). 
+- *It makes financial sense:* around 20% of working age adults in the UK are estimated to have some kind of permanent disability (Source: [Scope UK](https://www.scope.org.uk/media/disability-facts-figures)). The Financial Times estimates that 26% of their users have a permanent disability (Source: [Laura Carvajal, JSConf](https://www.youtube.com/watch?v=H4FzW9oFObs)). Also taking into account people who are [temporarily disabled](https://userway.org/blog/how-situational-disabilities-impact-us-all), failing to make our websites accessible can cause a significant reduction in real market share. 
+- *It's a technical requirement:* catering for all users complements our [browser support policy](../practices/graded-browser-support.md) (we aim to support all browsers), and our use of [progressive enhancement](../practices/progressive-enhancement.md). 
+- *It's the right thing to do:* we have a duty to cater for all users regardless of ability. If we know that people can encounter difficulties accessing our content (which we do), and we know that we can do things to facilitate that access (which we do), then not making something accessible is knowingly contributing to the oppression of people with disabilities.
+
+Accessible sites can be used by more people - including people with disabilities, people using mobile devices, older people, people with low literacy, people who are not fluent in the language of the site, people with low bandwidth connections to the Internet, people with older technologies, and new and infrequent web users.
+
+
+## VPATs
+
+A Voluntary Product Accessibility Template (VPAT) is a document which evaluates how accessible a particular product is [according to the Section 508 Standards](https://www.section508.gov/content/sell/vpat). 
+
+A VPAT MUST be produced by you or your team for every product or service that we make commercially available in the United States. If something changes, e.g. accessibility is improved for a particular criteria, you MUST update the VPAT to reflect the change. 
+
+You may use [our VPAT template](https://github.com/springernature/vpat) when creating this for your own products. 
+
+
+## How we conform
+
+### Standards we conform to
```

Comment:
Replace all the contents of this section with "Our target standards are WCAG 2.0 level AA".

Details on how we achieve this can be included in "Tools and techniques".